CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ Documents changes that result in:
 
 ## Unreleased
 
+## [0.26.0](https://github.com/raiden-network/raiden-contracts/releases/tag/v0.26.0) - 2019-07-11
+
+- [#1119](https://github.com/raiden-network/raiden-contracts/pull/1119) Changed the signature construction of monitoring reward proof so that a signature covers the whole reward proof
```

Comment:
```suggestion
- [#1119](https://github.com/raiden-network/raiden-contracts/pull/1119) Changed the signature construction of monitoring reward proof so that a signature covers the whole monitor request
```